arangod/Cluster/ClusterMethods.cpp

```diff
@@ -3449,11 +3626,14 @@ arangodb::Result hotBackupCoordinator(ClusterFeature& feature, VPackSlice const
          (payload.hasKey("label") && !payload.get("label").isString()) ||
          (payload.hasKey("timeout") && !payload.get("timeout").isNumber()) ||
          (payload.hasKey("allowInconsistent") &&
-          !payload.get("allowInconsistent").isBoolean()))) {
+          !payload.get("allowInconsistent").isBoolean()) ||
+         (payload.hasKey("force") &&
+          !payload.get("force").isBoolean()))) {
       return arangodb::Result(TRI_ERROR_BAD_PARAMETER, BAD_PARAMS_CREATE);
     }
 
-    bool force = !payload.isNone() && payload.get("allowInconsistent").isTrue();
+    bool allowInconsistent = !payload.isNone() && payload.get("allowInconsistent").isTrue();
```

Comment:
This can be simplified to `payload.get("allowInconsistent").isTrue()`, because this is also possible for a None slice and will return `false`.